examples/server-example.py

```diff
@@ -74,11 +72,26 @@ def multiply(parent, x, y):
     uri = "http://examples.freeopcua.github.io"
     idx = server.register_namespace(uri)
 
-    # get Objects node, this is where we should put our custom stuff
+
+    # create a new node type we can instantiate in our address space
+    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
```

Comment:
Upper case for class members is not part of pep5 but maybe something like:
Server.objects
Server. Server
Server. Basetype

Or:
Server.nodes. Objects
Server.nodes. Server
Or:
Server.shortcuts. Objects
Server.shortcuts. Server

Any better ideas?

On Wed, Jul 20, 2016, 17:55 Andrew <notifications@github.com> wrote:

> In examples/server-example.py
> <https://github.com/FreeOpcUa/python-opcua/pull/244#discussion_r71553048>:
>
> > @@ -74,11 +72,26 @@ def multiply(parent, x, y):
> >      uri = "http://examples.freeopcua.github.io"
> >      idx = server.register_namespace(uri)
> >
> > -    # get Objects node, this is where we should put our custom stuff
> > +
> > +    # create a new node type we can instantiate in our address space
> > +    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
>
> Either is fine for me. I was just thinking that it might be cleaner to get
> a few "constants" when the server is created. If you just got a reference
> to the base type node on init, we do not need more methods in the server
> class.
>
> However I don't know if that is a proper python convention.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeOpcUa/python-opcua/pull/244/files/88f2ae0f075c2248c3421c956ab06051cd3d1ec9..785f185239fb050ac90cca77296cff6d996c58ee#r71553048>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACcfzpNBfurAzSHw0U1fa1Kx44iIO2Rgks5qXkTVgaJpZM4JLHjh>
> .
>